python/Ice/greeter/client/main.py

```diff
@@ -2,26 +2,35 @@
 # Copyright (c) ZeroC, Inc.
 
 import Ice
+import asyncio
 import getpass
 import sys
 
 # Slice module VisitorCenter in Greeter.ice maps to Python module VisitorCenter.
 import VisitorCenter
 
-# Create an Ice communicator to initialize the Ice runtime. The communicator is destroyed at the end of the with block.
-with Ice.initialize(sys.argv) as communicator:
+async def main():
 
-    # GreeterPrx is a class generated by the Slice compiler. We create a proxy from a communicator and a "stringified
-    # proxy" with the address of the target object.
-    # If you run the server on a different computer, replace localhost in the string below with the server's hostname
-    # or IP address.
-    greeter = VisitorCenter.GreeterPrx(communicator, "greeter:tcp -h localhost -p 4061")
+    # Get the current event loop.
+    loop=asyncio.get_running_loop()
```

Comment:
I would not put it in a separate variable; I'd prefer a simpler `eventLoop=asyncio_get_running_loop()` in the initialize below.